core/src/apps/common/backup.py

```diff
@@ -1,25 +1,27 @@
 from typing import TYPE_CHECKING
 
-import storage.cache as storage_cache
```

Comment:
Is the change in import style from 
- `import storage.cache as storage_cache` and using `storage_cache.CONSTANT`
to
- `from storage.cache import CONSTANT` and using `CONSTANT`
ok, or is there a reason to NOT do it the way it is in the PR? @mmilata

Reply:
ack